### PR TITLE
Complete all CRUD API routes

### PR DESCRIPTION
**Pull request by kage1414**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
```
Comment (djian001) [inline .eslintrc.js:1]: What is this file for?

File: README.md [hunk 2]
```diff
@@ -94,6 +94,26 @@ DB_HOST={mysqlHost}
    - /api/book/{id}/related
        * returns an object containing related books
        * {byAuthor: [], byNarrator:[]}
+  POST:
```
Comment (djian001) [inline README.md:97]: great job keeping up with the README!

**Review by djian001 — APPROVED** [review]
Great job following the format of the legacy code! Curious as to why you used PUT vs PATCH for your update.

File: .eslintrc.js (hunk 1 above)
Comment (kage1414) [reply .eslintrc.js:1]: Eslint configuration. I don't have it setup globally

**Comment by kage1414** [comment]
> Curious as to why you used PUT vs PATCH for your update.

PUT was what was specified in gLearn. After reading some documentation it looks like PATCH would be used to update a part of a record, whereas PUT should be able to update the entire record or upsert it if it doesn't exist. Will probably stick to PUT for now

**Pull request merged**